src/passthrough/file_handle.rs

```diff
@@ -271,6 +298,7 @@ impl OpenableFileHandle {
     ///
     /// If `path` is empty, `reopen_dir` may be invoked to duplicate `dir` with custom
     /// `libc::open()` flags.
+    #[cfg(test)]
     pub fn from_name_at<F>(
```

Comment:
If it is test only, move it to the tests module?

Reply:
Or do we need it at all? The test is actually testing `from_name_at`.

Reply:
Ok, I will remove it.